src/functions-and-arrays.js

```diff
@@ -52,15 +109,32 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let arr2 = [];
+  if (arr.length === 0) {return null}
+  else {
+    for (i = 0; i < arr.length; i++) {
+    if (arr2.length === 0) {
+      arr2.push(arr[i])
+      }
+    else if (arr2.includes(arr[i]) === false) {arr2.push(arr[i])}
+    else continue
```

Comment:
the loop will continue on its own. It's not necessary to add another else block that says continue ^^